### ISSUE TITLE
Water levels with invalid solutions

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#603]
Infiltrate and UnderPressure seem to have slightly messed up solutions.

```bash
$ make test
...
    [junit] Testcase: All_staging_levels_work_and_have_winning_solutions took 2.066 sec
    [junit] 	Caused an ERROR
    [junit] Solution failed: world has stopped (state: LOST) but there are more solution commands at command 29 of solution 1 in water/Under-pressure:
...
To see: ./runrabbit swing -l water/Under-pressure -s1
...
   [junit] rabbitescape.engine.solution.SolutionExceptions$RanPastEnd: Solution failed: world has stopped (state: LOST) but there are more solution commands at command 29 of solution 1 in water/Under-pressure:
...
    [junit] Solution failed: tried to place a bash token at (2, 3) but a block was already there so it did not place at command 31 of solution 1 in water/Infiltrate:
...
    [junit] To see: ./runrabbit swing -l water/Infiltrate -s1
```

**Comment by colonelfazackerley** [comment]
Not spent enough time to be sure, but...

Infiltrate at 326b7ff works with HEAD. I think 71c6bf3 was wrong because it uses solutions from before a subtle timing change introduced for a bugfix. revert those 2 levels to 326b7ff, and I think we are good.

**Comment by andybalaam** [comment]
Thank you and sorry.  Fixed in b9845ba76fc1f37b61a23081ab9b8337bd854298 and 3ff1a8e8b7ed37cd412f491508b44e9b8f564f56, and I changed the default make target to run tests to help avoid me doing something like this in future.
